pkg/configuration/router.go

```diff
@@ -160,7 +160,7 @@ func uploadOrganizationLogo(logger log.Logger, repo Repository, bucketFactory st
 
 		file, _, err := r.FormFile("file")
 		if file == nil || err != nil {
-			logger.LogError(errMissingFile.Error(), err)
+			logger.LogErrorf("%s: %v", errMissingFile.Error(), err)
```

Comment:
You can do `loger.LogErrof("%w: %w", errMissingFile, err)`

Reply:
`fmt/errors.go`:
> It is invalid to include more than one %w verb